src/lib/select/select.spec.ts

```diff
@@ -527,6 +527,8 @@ describe('MdSelect', () => {
       fixture.detectChanges();
 
       const overlayPane = overlayContainerElement.children[0] as HTMLElement;
+      overlayPane.style.position = 'absolute';
```

Comment:
It's because it doesn't look like we import all of the styles in the unit tests. This means that the overlay is `position: static`. It wasn't an issue with the transform approach since they work on all of the positions.